sample/TimerTrigger-CSharp/function.json

```diff
@@ -3,7 +3,6 @@
         {
             "type": "timerTrigger",
             "schedule": "0 * * * * *",
-            "runOnStartup": true,
```

Comment:
Removing these from the samples since for Functions people generally don't need to do this.